### PR TITLE
installer.go amend some comment

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE

### CODE REVIEW

**Comment by jianglingxia** [comment]
ping @wojtek-t @deads2k

File: staging/src/k8s.io/apiserver/pkg/endpoints/installer.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ type action struct {
 	Path          string               // The path of the action
 	Params        []*restful.Parameter // List of parameters associated with the action.
 	Namer         handlers.ScopeNamer
-	AllNamespaces bool // true iff the action is namespaced but works on aggregate result for all namespaces
+	AllNamespaces bool // true if the action is namespaced but works on aggregate result for all namespaces
```
Comment (deads2k) [inline staging/src/k8s.io/apiserver/pkg/endpoints/installer.go]: I don't think this is a typo.

File: staging/src/k8s.io/apiserver/pkg/endpoints/installer.go [hunk 2]
```diff
@@ -841,7 +841,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	return &apiResource, nil
 }
 
-// This magic incantation returns *ptrToObject for an arbitrary pointer
+// The indirectArbitraryPointer returns *ptrToObject for an arbitrary pointer
```
Comment (deads2k) [inline staging/src/k8s.io/apiserver/pkg/endpoints/installer.go]: remove `The`

**Comment by jianglingxia** [comment]
yeah ,later I know iff  means if and only if ,and I have updated it ,thanks for your advices and PTAL

**Comment by jianglingxia** [comment]
ping @wojtek-t @deads2k

**Comment by jianglingxia** [comment]
/assign @cblecker
please review the pr thanks

**Comment by cblecker** [comment]
/unassign
Please do not randomly assign people. Your PR is already assigned, and your reviewer is responsive.

On a side note, please update the commit message to be meaningful.

**Comment by wojtek-t** [comment]
> Please do not randomly assign people. 

+100

> On a side note, please update the commit message to be meaningful.

+1

/ok-to-test

**Comment by wojtek-t** [comment]
Especially since this isn't any urgent fix or sth like that.

**Pull request merged**